Shared/Samples/Snap geometry edits/SnapGeometryEditsView.SnapSettingsView.swift

```diff
@@ -61,9 +80,9 @@ extension SnapGeometryEditsView {
                 }
             }
             .onAppear {
-                // Enables snapping in the current geometry editor.
-                model.geometryEditor.snapSettings.isEnabled = true
```

Comment:
This fixes a bug where the "Snapping" toggle would enable every time the sheet was opened instead of saving state. Now, `isEnabled` is just set once when the `GeometryEditor` is created in the model.